package.go

```diff
@@ -67,7 +68,7 @@ func (pkg *Package) GeneratePackage(dir string) error {
 }
 
 func (pkg *Package) generateFile(file, dir string) error {
-	out, err := os.Create(filepath.Join(dir, file+".go"))
+	out, err := os.Create(filepath.Join(dir, file + ".go"))
```

Comment:
This is not `gofmt`ed, that's why it got reverted in https://github.com/go-gl/glow/commit/0d9b82783d5f4b9dafacfcc53da05b2f27e166b1.